app/views/admin/educators/_navigation.html.erb

```diff
@@ -10,6 +10,7 @@ as defined by the routes in the `admin/` namespace
 <% if action_name == "show" || action_name == "edit" %>
   <nav class="navigation" role="navigation">
     <% Administrate::Namespace.new(namespace).resources.each do |resource| %>
+      <% next unless resource.name == :educators %>
```

Comment:
Administrate reflects on the routes in the admin namespace, and assumes it can map all those 1:1 to controllers and models.  Since it can't, this whitelists it to only do this for the educator model.